src/dxgi/dxgi_monitor.h

```diff
@@ -55,4 +57,32 @@ namespace dxvk {
   uint32_t GetMonitorFormatBpp(
           DXGI_FORMAT             Format);
 
+  /**
+   * \brief Converts two display modes
+   */
+  inline void ConvertDisplayMode(
+    const wsi::WsiMode&          WsiMode,
+          DXGI_MODE_DESC1*       pDxgiMode) {
+    pDxgiMode->Width            = WsiMode.width;
+    pDxgiMode->Height           = WsiMode.height;
+    pDxgiMode->RefreshRate      = DXGI_RATIONAL{ WsiMode.refreshRate.numerator, WsiMode.refreshRate.denominator };
+    pDxgiMode->Format           = DXGI_FORMAT_R8G8B8A8_UNORM_SRGB; // FIXME
+    pDxgiMode->ScanlineOrdering = DXGI_MODE_SCANLINE_ORDER_PROGRESSIVE;
```

Comment:
Could set `DXGI_MODE_SCANLINE_ORDER_UPPER_FIELD_FIRST` if the mode at hand is interlaced